Source/WebCore/rendering/style/StyleGeneratedImage.h

```diff
@@ -56,6 +56,7 @@ class StyleGeneratedImage : public StyleImage {
     bool imageHasRelativeHeight() const final { return !m_fixedSize; }
     bool usesImageContainerSize() const final { return !m_fixedSize; }
     void setContainerContextForRenderer(const RenderElement&, const FloatSize& containerSize, float) final { m_containerSize = containerSize; }
+    bool imageHasNaturalDimensions() const final { return !usesImageContainerSize(); }
```

Comment:
It's great that we are starting to use the spec language here. We should try to replace the other terminology in StyleImage to match the CSS Images spec language (https://drafts.csswg.org/css-images-3/#sizing-terms) as much as we can.